spec/javascripts/ga_cross_domain_form_tracking_spec.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-env jasmine, jquery */
+/* global attachCrossDomainTrackerToInput */
```

Comment:
Seems like there should be a better way to quiet the liner here...
Am i supposed to be importing / destrcturing these into my tests... they are globally available?

Fees like fighting the linter?

Reply:
What we should probably be doing is adding `attachCrossDomainTrackerToInput` to the `window.GOVUK` object. Which would involve more work. Sorry. But then you wouldn't have to do this.

Reply:
So i think i've done that here: https://github.com/alphagov/finder-frontend/pull/2289/commits/8eede1974f3e8127018d7f5725df1013a63f1d70